extensions/timeular/src/useTodayReport.ts

```diff
@@ -23,18 +23,37 @@ export const useTodayReport = (tracking: Tracking | null, activity?: Activity) =
     if (reportIsLoading) {
       return;
     }
+    const ms = Date.now() - new Date(tracking?.startedAt + "Z").getTime();
 
     const markStart = Date.now();
 
+    const totalBookedTime = entries.reduce((acc, val) => {
+      if (!val.duration.stoppedAt || !val.duration.startedAt) {
+        // Skip entries that are still running (no stoppedAt)
+        return acc;
+      }
```

Comment:
In which scenarios `val.duration.startedAt` isn't there?

Reply:
There probably is not a scenario where `val.duration.startedAt` would be missing. I supposed I just threw it in there __in case__ 🤷🏻‍♂️. 

I think it can probably be removed. 

We should still leave the `val.duration.stoppedAt`. We need to ensure that it exists, otherwise  it is the currently running task.